joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h

```diff
@@ -319,4 +321,81 @@ class HardwareInterfaceAdapter<hardware_interface::EffortJointInterface, State>
   std::vector<hardware_interface::JointHandle>* joint_handles_ptr_;
 };
 
+/**
+ * \brief Adapter for a PosVelAcc hardware interface. Forwards desired positions, velocities
+ * and accelerations as commands.
+ */
+template <class State>
+class HardwareInterfaceAdapter<hardware_interface::PosVelAccJointInterface, State>
+{
+public:
+  HardwareInterfaceAdapter() : joint_handles_ptr_(0) {}
+
+  bool init(std::vector<hardware_interface::PosVelAccJointHandle>& joint_handles, ros::NodeHandle& controller_nh)
+  {
+    // Store pointer to joint handles
+    joint_handles_ptr_ = &joint_handles;
+
+    return true;
+  }
+
+  void starting(const ros::Time& time) {}
```

Comment:
Please implement this method like it's done for the position joint ([source](https://github.com/sachinchitta/ros_controllers/blob/add_posvelacc_interface/joint_trajectory_controller/include/joint_trajectory_controller/hardware_interface_adapter.h#L108-L117))